modules/Date.js

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from './luxoncode.js';
+
+function currentTime() {
+  const date = document.getElementById('date');
+
+  setInterval(() => {
+    const now = DateTime.now();
+    date.innerHTML = now.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  }, 1000);
+}
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.